CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+
+- Extended `SignedInAccount` GraphQL type to include `customer_ids` field for
```

Comment:
@octoaide

Could you please revise the changelog entry related to `SignedInAccount` as follows:
```md
Added support for customer ids information in `SignedInAccountList` GraphQL API via the `SignedInAccount` type.
```
includes implementation motivation from the UI perspective, which may not be necessary in the server-side changelog. The revised version focuses on what was changed in the API itself, which should be sufficient for backend documentation.